index.js

```diff
@@ -136,9 +143,34 @@ instructions:
       or anything like that.
 `;
 
-const removeColorsFormat = winston.format((info) => {
-    info.message = stripAnsi(info.message);
-    return info;
+const testPlanSchema = z.object({
+    arrayOfSpecs: z.array(z.string()),
```

Comment:
GPT-4o will only accept JSON/zod schemas where the base type is an object. I originally tried `const testPlanSchema = z.array(z.string());` and it threw an error saying that. So this object shape ensures cross-compatability.